_schemaV2.graphql

```diff
@@ -9365,6 +9365,13 @@ enum ExhibitionPeriodFormat {
   SHORT
 }
 
+# A list of marketing collection categories to explore
+type ExploreByMarketingCategories {
```

Comment:
Does this type represent 1 category or a list of categories?

Reply:
good catch, it should be a single one, on it 🫡